dockeradapter/dockeradapter.go

```diff
@@ -123,10 +124,23 @@ func ImageExists(ctx context.Context, client ImageInspector, imageName string) b
 func New() (Client, error) {
 	concreteClient := new(concreteDockerClient)
 	logging.Stderr("[Docker Adapter] Creating container runtime client: Docker")
+	ctx := context.Background()
 	dockerAPIClient, err := client.NewEnvClient()
 	if err != nil {
-		return nil, fmt.Errorf("[Docker Adapter] Unable to initialise container runtime type: Docker, error: %s", err)
+		return nil, fmt.Errorf("[Docker Adapter] Unable to initialize container runtime type: Docker, error: %s", err)
 	}
+
+	serverVersion, _ := dockerAPIClient.ServerVersion(ctx)
+	clientVersion := dockerAPIClient.ClientVersion()
+	clientAPIVersion, _ := version.NewVersion(clientVersion)
+	serverAPIVersion, _ := version.NewVersion(serverVersion.APIVersion)
```

Comment:
This might be nil if it has a parse error, think thats where the panic is occurring.

*edit* yeah need to check whats going on here:
```
[lumogon] 2017/06/07 20:08:48.948020 [Analytics] Initializing Google Analytics: scan
[lumogon] 2017/06/07 20:08:48.948096 [Docker Adapter] Creating container runtime client: Docker
[lumogon] 2017/06/07 20:08:48.958865 DEBUG: ServerVersion
[lumogon] 2017/06/07 20:08:48.958874 DEBUG: clientVersion 1.27
[lumogon] 2017/06/07 20:08:48.958943 DEBUG: Error thrown parsing serverAPIVersion: Malformed version:
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x6f0921]
```